scio-core/src/main/scala/com/spotify/scio/ScioResult.scala

```diff
@@ -176,4 +178,8 @@ class ScioResult private[scio] (val internal: PipelineResult,
     aggregators.getOrElse(acc.name, Nil)
```

Comment:
Yeah maybe let's do what you suggest stub the aggregators in template staging + print warning maybe, this would allow people to just create templates without code change, while still be informed that it's only the computation DAG that is executed.